### PR TITLE
l10n: uk: update core translation

### PR DESCRIPTION
**Pull request by arkid15r**

### CODE REVIEW

File: po/uk.po [hunk 1]
```diff
@@ -608,155 +608,156 @@ msgstr "використовувати режим накладення"
...
 
 #: builtin/clone.c:105 builtin/clone.c:107
 msgid "create a bare repository"
-msgstr ""
+msgstr "створити порожній репозиторій"
 
 #: builtin/clone.c:109
 msgid "create a mirror repository (implies bare)"
-msgstr ""
+msgstr "створити дзеркальний репозиторій (включає опцію bare)"
```
Comment (kasya) [inline po/uk.po:627]:
```suggestion
msgstr "створити дзеркальний репозиторій (мається на увазі порожній репозиторій)"
```

File: po/uk.po [hunk 2]
```diff
@@ -608,155 +608,156 @@ msgstr "використовувати режим накладення"
...
 
 #: builtin/clone.c:140
 msgid "revision"
-msgstr ""
+msgstr "ревізія"
 
 #: builtin/clone.c:141
 msgid "deepen history of shallow clone, excluding rev"
-msgstr ""
+msgstr "поглибити історію поверхневого клону, виключаючи rev"
```
Comment (kasya) [inline po/uk.po]:
```suggestion
msgstr "поглибити історію поверхневого клону, за винятком rev"
```

File: po/uk.po [hunk 3]
```diff
@@ -608,155 +608,156 @@ msgstr "використовувати режим накладення"
...
 
 #: builtin/clone.c:150
 msgid "key=value"
-msgstr ""
+msgstr "ключ=значення"
 
 #: builtin/clone.c:151
 msgid "set config inside the new repository"
-msgstr ""
+msgstr "встановити конфіг всередині нового репозиторія"
```
Comment (kasya) [inline po/uk.po]:
```suggestion
msgstr "встановити конфігурацію всередині нового репозиторія"
```

**Review by kasya — COMMENTED** [review]
LGTM

File: po/uk.po (hunk 1 above)
Comment (arkid15r) [reply po/uk.po:627]: I'm not sure if the resulting repository would be empty here. My understanding is that it just uses --bare.

**Pull request merged**